networkit/community.pyx

```diff
@@ -1080,7 +1080,7 @@ def detectCommunities(G, algo=None, inspect=True):
 	algo.run()
 	zeta = algo.getPartition()
 	t.stop()
-	print("{0} detected communities in {1} [s]".format(algo.toString(), t.elapsed))
+	print("Communities detected in {:.5f} [s]".format(t.elapsed))
```

Comment:
Even though `toString()` creates runtime-errors for many algorithms, deprecation and removal should be two steps.

Reply:
I think that removing usages of a newly deprecated feature is fine.

Reply:
Yes, this change should be applied to prevent deprecation warnings when building NetworKit itself.

(But Fabian is right that the function definition should not be removed.)

Reply:
That makes sense of course.